README.md

```diff
@@ -1,12 +1,35 @@
-# Node SDK for Crowdin API v2
+[<p align="center"><img src="https://support.crowdin.com/assets/logos/crowdin-dark-symbol.png" data-canonical-src="https://support.crowdin.com/assets/logos/crowdin-dark-symbol.png" width="200" height="200" align="center"/></p>](https://crowdin.com)
 
-Node.js client for integration with Crowdin API v2.
+# Crowdin JavaScript client
 
-For more about Crowdin API v2 see [Crowdin API](https://support.crowdin.com/enterprise/api/)
+The Crowdin JavaScript client is a lightweight interface to the Crowdin API v2 that works in both Node.js and browser environments. It provides common services for making API requests.
+
+For more about Crowdin API v2 see [Crowdin API v2](https://support.crowdin.com/enterprise/api/) documentation.
+
+
+## Table of Contents
+* [Requirements](#requirements)
+* [Dependencies](#dependencies)
+* [Installation](#installation)
+* [Quick Start](#quick-start)
+* [Contribution](#contribution)
+* [Seeking Assistance](#seeking-assistance)
+* [License](#license)
+
+## Requirements
+* typescript 3.5.2
+* tslint 5.18.0
+* ts-jest 24.0.2
+* (optional) jest 24.8.0
+* (optional) jest-circus 24.8.0
+
+## Dependencies
+* Axios 0.19
```

Comment:
Well...in `dependencies` there is also types for Node.js. Probably we can move them into `devDependencies`. But we need to check it.